processor/configuration/src/main/java/com/redhat/service/smartevents/processor/GatewayConfiguratorService.java

```diff
@@ -10,4 +10,14 @@ public interface GatewayConfiguratorService {
     String getBridgeEndpoint(String bridgeId, String customerId);
 
     String getConnectorTopicName(String processorId);
+
+    String getBootstrapServers();
+
+    String getClientId();
+
+    String getClientSecret();
+
+    String getSecurityProtocol();
+
+    String getBridgeErrorTopicName(String testBridgeId);
```

Comment:
what is the purpose of `getBridgeErrorTopicName` ? I see it is injected everywhere but I don't see where it ends up

Reply:
it's set as `dead-letter-queue.topic` in the outbound channel